corehq/apps/change_feed/management/commands/validate_kafka_pillow_checkpoints.py

```diff
@@ -30,7 +31,8 @@ def handle(self, **options):
 def validate_checkpoints(print_only):
 
     for pillow in get_all_pillow_instances():
-        if isinstance(pillow.get_change_feed(), KafkaChangeFeed):
+        if (pillow.pillow_id in getattr(settings, 'ACTIVE_PILLOW_NAMES', [pillow.pillow_id]) and
+           isinstance(pillow.get_change_feed(), KafkaChangeFeed)):
```

Comment:
this indentation was hard for me to parse. Could this line be indented more? or perhaps bringing the `and` to the next line. Something similar to:

```python
# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```

as described here: https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
Sure